Embark.colorscheme

```diff
@@ -0,0 +1,111 @@
+[Background]
+Color=30,28,49
+RandomHueRange=360
+RandomSaturationRange=100
+
+[BackgroundFaint]
+Color=30,28,49
+RandomHueRange=360
+RandomSaturationRange=100
+
+[BackgroundIntense]
+Color=30,28,49
+RandomHueRange=360
+RandomSaturationRange=100
+
+[Color0]
+Color=30,28,49
+
+[Color0Faint]
+Color=30,28,49
+
+[Color0Intense]
+Color=88,82,115
+
+[Color1]
+Color=244,143,177
+
+[Color1Faint]
+Color=244,143,177
+
+[Color1Intense]
+Color=240,46,110
+
+[Color2]
+Color=161,239,211
+
+[Color2Faint]
+Color=161,239,211
+
+[Color2Intense]
+Color=98,209,150
+
+[Color3]
+Color=255,230,179
+
+[Color3Faint]
+Color=255,230,179
+
+[Color3Intense]
+Color=242,180,130
+
+[Color4]
+Color=145,221,255
+
+[Color4Faint]
+Color=145,221,255
+
+[Color4Intense]
+Color=101,178,255
+
+[Color5]
+Color=212,191,255
+
+[Color5Faint]
+Color=212,191,255
+
+[Color5Intense]
+Color=163,122,204
+
+[Color6]
+Color=135,223,235
+
+[Color6Faint]
+Color=135,223,235
+
+[Color6Intense]
+Color=99,242,241
+
+[Color7]
+Color=203,227,231
+
+[Color7Faint]
+Color=203,227,231
+
+[Color7Intense]
+Color=138,136,157
+
+[Foreground]
+Color=203,227,231
+RandomHueRange=360
+RandomSaturationRange=100
+
+[ForegroundFaint]
+Color=203,227,231
+RandomHueRange=360
+RandomSaturationRange=100
+
+[ForegroundIntense]
+Color=203,227,231
+RandomHueRange=360
+RandomSaturationRange=100
+
+[General]
+Anchor=0.5,0.5
+Blur=false
```

Comment:
**Question -** Is defining blur required for a Konsole theme

Ideally the theme is just a palette and doesn't cause any other changes. I think users should be able to set whatever opacity, blur, etc options they want. Is the definition of these options necessary or can it be dropped?

Reply:
they can be safely dropped, but it makes no difference as Konsole would init them on the exact values set there anyway (actually why they're even in the file, created the scheme with Konsole's own editor and it set those values there)